### PR TITLE
feat(UI:PageTitle): Show selected Project/Component Name in Browser Tab

### LINKED ISSUE #140: Show selected Project/Component Name in Browser Tab / Document Title
<a href="https://github.com/AndreasSchirm"></img></a> **Issue by [AndreasSchirm](https://github.com/AndreasSchirm)**
_Tuesday Sep 05, 2017 at 08:32 GMT_
_Originally opened as https://github.com/sw360/sw360portal/issues/550_

----

## Expected Behavior
A browser tab should contain the actual selected project or component.

## Current Behavior
img_2017 09 05_ 10 26 35
It is always the generic text `Projects - SW360` ot `Components - SW360`.

## Possible Solution
Set html title to the corresponding Project/Component.

### PR DESCRIPTION
**Pull request by nutanv1contr**
#### Summary:
Show selected Project/Component Name in Browser Tab
#### Changes:
Added project or component name to page title.Now on selecting project or component the page title for Project will be "(ProjectName) : Projects - Liferay " and similarly on selecting of component the title will be ""(ComponentName) : Components - Liferay"
#### Steps to test:
1. Go to Projects tab and click on any of the project from the list.
2. Go to Components tab and select any of the existing component.
#### Expected result:
1.Selected Project or Component  name should be coming on page title.

Closes #140

### CODE REVIEW

File: frontend/sw360-portlet/src/main/webapp/html/components/edit.jsp [hunk 1]
```diff
@@ -145,6 +145,7 @@
 </core_rt:if>
 
 <script>
+    document.title = "${component.name}:" + document.title
```
Comment (mcjaeger) [inline frontend/sw360-portlet/src/main/webapp/html/components/edit.jsp]: how about adding a space after the double points

**Comment by mcjaeger** [comment]
(from discussion) it is more about consistency, eitherusing hyphen or double point

**Comment by nutanv1contr** [comment]
Thanks @mcjaeger for reviewing the changes.I have pushed the required changes.

**Comment by nutanv1contr** [comment]
Thank you @imaykay  for reviewing the changes. Yes semicolon is needed and I missed that at few places. I have updated the same and pushed the latest changes.As you said for displaying project or component name on page title I am preferring to go with client side coding.

**Comment by mcjaeger** [comment]
@imaykay please consider approving

**Pull request merged**